### PR TITLE
Added Extend cross-linking

### PR DESCRIPTION
**Pull request by rkhoriander**
Per docs-requests:
https://auth0.slack.com/archives/C1CCJPKAM/p1537205637000100

### CODE REVIEW

**Comment by rkhoriander** [comment]
https://auth0-docs-content-pr-6643.herokuapp.com/docs/rules/current
https://auth0-docs-content-pr-6643.herokuapp.com/docs/hooks
https://auth0-docs-content-pr-6643.herokuapp.com/docs/extensions
https://auth0-docs-content-pr-6643.herokuapp.com/docs/topics/extensibility

File: articles/extensions/index.md [hunk 1]
```diff
@@ -65,3 +65,14 @@ Auth0 provides the following pre-defined extensions, and they are available for
...
+
+---
+
+<img align="right" src="/media/articles/extensions/extend-extensions.svg"> 
+
+## Extensions in your SaaS
+
+If you want Auth0 extensibility capabilities in your product, try Auth0 Extend--an embedded scripting environment for enabling customization and integration of your SaaS.
+
+[Try Extend &#9658;](https://goextend.io/)
```
Comment (tjanczuk) [inline articles/extensions/index.md]: Please use `https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=extensions-paragraph` for URL

File: articles/extensions/index.md [hunk 2]
```diff
@@ -65,3 +65,14 @@ Auth0 provides the following pre-defined extensions, and they are available for
...
 ### Create a SSO dashboard with multiple enterprise applications 
 - [SSO Dashboard Extension](extensions/sso-dashboard)
+
+---
+
+<img align="right" src="/media/articles/extensions/extend-extensions.svg"> 
+
+## Extensions in your SaaS
+
+If you want Auth0 extensibility capabilities in your product, try Auth0 Extend--an embedded scripting environment for enabling customization and integration of your SaaS.
```
Comment (tjanczuk) [inline articles/extensions/index.md]: If you like the capabilities of Auth0 Extensions and would like to provide a similar experience in your own SaaS product, try Extend by Auth0. Extend is an embedded scripting environment for enabling integration and customization of your SaaS.

File: articles/hooks/index.md [hunk 3]
```diff
@@ -23,6 +23,20 @@ useCase: extensibility-hooks
 
 When using [Database Connections](/connections/database), Hooks allow you to customize the behavior of Auth0 using Node.js code that is executed against extensibility points (which are comparable to webhooks that come with a server). Hooks allow you modularity when configuring your Auth0 implementation, and extend the functionality of base Auth0 features.
 
+---
+
+<img align="right" src="/media/articles/hooks/extend-hooks.svg"> 
+
+## Hooks in your SaaS
+
+If you want to create and customize Hooks in your product, try Auth0 Extend--an embedded scripting environment for enabling customization and integration of your SaaS.
```
Comment (tjanczuk) [inline articles/hooks/index.md]: If you like the capabilities of Hooks and would like to provide a similar experience in your own SaaS product, try Extend by Auth0. Extend is an embedded scripting environment for enabling integration and customization of your SaaS.

File: articles/rules/current/index.md [hunk 4]
```diff
@@ -13,32 +13,44 @@ useCase:
...
 * Modify tokens: Change the returned __scopes__ of the Access Token and/or add claims to it, and to the ID Token.
 
-## Video: Using rules
+---
+
+<img align="right" src="/media/articles/rules/extend-rules.svg"> 
+
+## Auth0 Rules in your SaaS
+
+If you want to create and customize Rules in your product, try Auth0 Extend--an embedded scripting environment for enabling customization and integration of your SaaS.
```
Comment (tjanczuk) [inline articles/rules/current/index.md]: If you like the capabilities of Rules and would like to provide a similar experience in your own SaaS product, try Extend by Auth0. Extend is an embedded scripting environment for enabling integration and customization of your SaaS.

File: articles/topics/extensibility.md [hunk 5]
```diff
@@ -46,3 +46,13 @@ Use Extensions to:
...
 - [Deploy rules, and other scripts from external repositories](/extensions#deploy-hosted-pages-rules-and-database-connections-scripts-from-external-repositories)
 - [and more](/extensions#what-types-of-actions-can-i-do-with-extensions-)
+
+---
+
+<img align="right" src="/media/articles/topics/extend-extensibility.svg"> 
+
+## Extensibility in your SaaS
+
+If you want Auth0 extensibility capabilities in your product, try Auth0 Extend--an embedded scripting environment for enabling customization and integration of your SaaS.
```
Comment (tjanczuk) [inline articles/topics/extensibility.md]: If you like the Auth0 extensibility capabilities and would like to provide a similar experience in your own SaaS product, try Extend by Auth0. Extend is an embedded scripting environment for enabling integration and customization of your SaaS.

File: articles/hooks/index.md [hunk 6]
```diff
@@ -23,6 +23,20 @@ useCase: extensibility-hooks
...
+
+[Try Extend &#9658;](https://goextend.io/)
```
Comment (tjanczuk) [inline articles/hooks/index.md]: Please use `https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=hooks-paragraph` for URL

File: articles/rules/current/index.md [hunk 7]
```diff
@@ -13,32 +13,44 @@ useCase:
...
+
+[Try Extend &#9658;](https://goextend.io/)
```
Comment (tjanczuk) [inline articles/rules/current/index.md]: Please use `https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=rules-current-paragraph` for URL

File: articles/topics/extensibility.md [hunk 8]
```diff
@@ -46,3 +46,13 @@ Use Extensions to:
...
+
+[Try Extend &#9658;](https://goextend.io/)
```
Comment (tjanczuk) [inline articles/topics/extensibility.md]: Please use `https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=extensibility-paragraph` for URL

File: articles/extensions/index.md (hunk 1 above)
Comment (rkhoriander) [reply articles/extensions/index.md]: Done.

File: articles/extensions/index.md (hunk 2 above)
Comment (rkhoriander) [reply articles/extensions/index.md]: Done.

File: articles/hooks/index.md (hunk 3 above)
Comment (rkhoriander) [reply articles/hooks/index.md]: Done.

File: articles/hooks/index.md (hunk 6 above)
Comment (rkhoriander) [reply articles/hooks/index.md]: Done.

File: articles/rules/current/index.md (hunk 4 above)
Comment (rkhoriander) [reply articles/rules/current/index.md]: Done.

File: articles/rules/current/index.md (hunk 7 above)
Comment (rkhoriander) [reply articles/rules/current/index.md]: Done.

File: articles/topics/extensibility.md (hunk 8 above)
Comment (rkhoriander) [reply articles/topics/extensibility.md]: Done.

File: articles/topics/extensibility.md (hunk 5 above)
Comment (rkhoriander) [reply articles/topics/extensibility.md]: Done.

**Review by khorne3 — DISMISSED** [review]

**Review by khorne3 — APPROVED** [review]

**Pull request merged**